### PR TITLE
oop exercise task

### PR DESCRIPTION
**Pull request by vitaliykravchyk**
### A-Tiny-JS-World-2

[Demo](https://vitaliykravchyk.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/vitaliykravchyk/frontend-2019-homeworks/tree/a-tiny-js-world-2/submissions/vitaliykravchyk/a-tiny-js-world-2)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/vitaliykravchyk/a-tiny-js-world-2/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+let Creature = function(species, name, hands, legs, gender, phrase){
+    this.species = species;
+    this.name = name;
+    this.hands = hands;
+    this.legs = legs;
+    this.gender = gender;
+    this.phrase = phrase;
+    this.friendList = [];
+}
+Creature.prototype.friends = function(...friend){
```
Comment (AMashoshyna) [inline submissions/vitaliykravchyk/a-tiny-js-world-2/index.js]: Most often functions and methods are called with verbs. How about `addFriends`?

File: submissions/vitaliykravchyk/a-tiny-js-world-2/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+const cat = new Creature('cat', 'Jinny', 0, 4, 'female', 'Meeeeooow');
+const catWomen = new Creature('catwoman', 'Emma', 2, 2, 'female', cat.phrase);
+
+man.friends(women, dog);
+women.friends(man, cat);
+cat.friends(women, catWomen);
+dog.friends(man);
+catWomen.friends(cat);
+function info(obj){
+    return obj.species + ';' + obj.name + ';' + obj.hands + ';' + obj.legs + ';' + obj.gender + ';' + obj.phrase + ';' + obj.friendList;
```
Comment (AMashoshyna) [inline submissions/vitaliykravchyk/a-tiny-js-world-2/index.js]: same here. `getInfo` or `getPrintableInfo` maybe?

**Comment by vitaliykravchyk** [comment]
Fixed

**Comment by AMashoshyna** [comment]
Thanks, I will merge it then. Well done!

**Pull request merged**